### PR TITLE
ModLens loading on Linux & lowercasing minimappanel.xml replacements

### PR DESCRIPTION
**Pull request by RustyD**
Update modinfo with correct casing for files.

This is based on @the-m4a original branch to try and fix some issues with mixed case files not working with Linux, and no minimap appearing.

I've run a quick test with Gathering Storm, and can now see a minimap (which isn't there under 'master' branch). The city banner is still missing though.

This PR partially addresses issue #36

### CODE REVIEW

**Comment by the-m4a** [comment]
Out of curiousity, does the map also show if you rename Integrations/ML/UI/minimappanel.lua and .xml to MinimapPanel.lua/.xml ?  

I'm wondering if the issue with those specific files isn't that the expansions were not all lowercased, rather that the original replacement did not match the casing style of the file it was replacing.

**Comment by RustyD** [comment]
I quickly create this commit https://github.com/RustyD/CQUI_Community-Edition/commit/502ec426f748c47833dda01670c95e396219ae5a on this branch https://github.com/RustyD/CQUI_Community-Edition/tree/linux-issues-36-alt to test that out as per your suggestion @the-m4a 

The minimap _was_ visible, but I only had 10(?) out of the 18(?) lenses available.

File: Integrations/ML/UI/Panels/modallenspanel.lua [hunk 1]
```diff
@@ -9,7 +9,18 @@ include( "CQUICommon.lua" );
 -- ===========================================================================
 
 g_ModLensModalPanel = {} -- Populated by ModLens_*.lua scripts
-include( "ModLens_", true )
+--include( "ModLens_", true )
+-- Need to do these individually, instead of using the include("ModLens_", true), because it appears on Linux ModLens_Builder_Config_Default.lua would get loaded before ModLens_Builder.lua (when ModLens_Builder.lua includes _Builder_Config_Default).
+-- Casing also matters for these filenames with Linux: these ModLens_ files are checked in to the repo with the casing seen below.
+include("ModLens_Archaeologist.lua");
```
Comment (Infixo) [inline Integrations/ML/UI/Panels/modallenspanel.lua:15]: It cannot be like this. Our goal is to make ML work on Linux with the default features. And the key feature is the ability to load multiple lenses, also for compatibility with other mods. So, such iterated list is a no-go.

Comment (RustyD) [reply Integrations/ML/UI/Panels/modallenspanel.lua:15]: Ok, not going to argue with that. I was really just trying to see what would/wouldn't work from the original branch rebased to master.
Having the minimap without all the extra lenses is a step forward from having no minimap and no lenses though!  :-)

Comment (Infixo) [reply Integrations/ML/UI/Panels/modallenspanel.lua:15]: Well, you raised a PR, hence the comment...

**Pull request closed without merging**